user-manager/src/main/resources/application.yaml

```diff
@@ -0,0 +1,15 @@
+spring:
+  flyway:
+    schemas: users
+  jpa:
+    show-sql: true
+    properties:
+      hibernate:
+        dialect: org.hibernate.dialect.PostgreSQLDialect
+        format_sql: true
+        default_schema: users
+    hibernate:
+      ddl-auto: update
+server:
+  error:
+    include-message: always
```

Comment:
I don't think it's a good idea to log everything in prod. But it's worth to do it locally. So, move this config to local one.
https://dev.to/flippedcoding/difference-between-development-stage-and-production-d0p